R/lint.R

```diff
@@ -382,7 +382,7 @@ reorder_lints <- function(lints) {
 #' Create a `lint` object
 #' @param filename path to the source file that was linted.
 #' @param line_number line number where the lint occurred.
-#' @param column_number column number where the lint occurred.
+#' @param number column number where the lint occurred.
```

Comment:
is that intentional? this would be a user-facing API change? I guess this is by mistake...